src/renderer/components/settings/TrayIconImagePicker.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { Forms, Switch, Toasts } from "@vencord/types/webpack/common";
+import { Settings } from "renderer/settings";
+
+import { SettingsComponent } from "./Settings";
+
+export const TrayIconImagePicker: SettingsComponent = ({ settings }) => {
+    return (
+        <>
+            <div id="tray-setting">
+                <div id="colLeft">
+                    <Switch
+                        key="tray"
+                        value={Settings.store.tray ?? false}
+                        onChange={v => (Settings.store.tray = v)}
+                        note={"Add a tray icon for Vesktop"}
+                    >
+                        {"Tray Icon"}
+                    </Switch>
+                </div>
+                <div id="colMiddle">
+                    <Forms.FormText>
+                        <a
+                            href="about:blank"
+                            onClick={e => {
+                                e.preventDefault();
+                                settings.trayIconPath = void 0;
+                            }}
+                        >
+                            Reset
+                        </a>
+                    </Forms.FormText>
+                </div>
+                <div id="colRight">
+                    <div className="tray-icon-wrap">
+                        <img
+                            className="tray-icon-image"
+                            src={VesktopNative.tray.getTrayIcon()}
+                            alt="hello"
+                            width="48"
+                            height="48"
+                        ></img>
+                        <input
+                            className="edit-button"
+                            type="image"
+                            src="https://cdn.discordapp.com/attachments/895550066453012480/1236925384482619433/I1oxwE7.png?ex=6639c808&is=66387688&hm=5c5f47a0c06b2e0580fd5494386bfbeebc001c20e3fd64f26783f360b12162ed&"
```

Comment:
this cdn link will expire and break

Reply:
oh you said this already! well it's a bit tough to make it permanent, you could upload it as a file and use the github raw link

Reply:
Changed to a github link as suggested. It is linked to Vencord/Vesktop so it will show correctly if merged